data_pipeline/semantic_scholar/semantic_scholar_recommendation_pipeline.py

```diff
@@ -138,17 +146,20 @@ def get_recommendation_response_json_from_api(  # pylint: disable=too-many-argum
     extended_provenance = {
         **(provenance or {}),
         'list_key': excludable_list_with_meta.list_key,
-        'list_meta': excludable_list_with_meta.list_meta
+        'list_meta': excludable_list_with_meta.list_meta,
+        'item_list': excludable_list_with_meta.json_data
     }
-    return get_response_json_with_provenance_from_api(
-        url,
-        params=params,
-        method='POST',
-        json_data=json_data,
-        provenance=extended_provenance,
-        session=session,
-        raise_on_status=False,
-        progress_message=progress_message
+    return get_recursive_json_compatible_value(
```

Comment:
this was a "quick fix". Not sure what the right level is for calling the function, maybe it should be part of `load_given_json_list_data_from_tempdir_to_bq`.